packages/react-scripts/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 _Nothing yet_
 
+### Changed
```

Comment:
Nope it shouldn't affect anything, but it's a more aggressive optimizer so it could introduce issues.